C/test/yahdlc_test.cpp

```diff
@@ -39,6 +39,10 @@ BOOST_AUTO_TEST_CASE(yahdlcTestGetDataInvalidInputs) {
   unsigned int recv_length = 0;
   char frame_data[8], recv_data[8];
 
+  ret = yahdlc_get_data_with_state(NULL, &control, frame_data, sizeof(frame_data),
```

Comment:
Can you please add a unit test not only for testing invalid input? Coverage should remain the same